### PR TITLE
Add search endpoint

### PR DESCRIPTION
**Pull request by jpmartinspt**
## Done
Added a new endpoint that searches for a term in topics of the `doc` category.

## QA
- `./run`
- `http://0.0.0.0:8030/search?term=<try_me>`
- Try a couple of search terms and make sure we get appropriate results. Category should be 15 for all the returned topics.

### CODE REVIEW

**Comment by Lukewh** [comment]
QA +1 though it seems that post titles aren't returned? Is this expected?

**Comment by jpmartinspt** [comment]
@Lukewh Yeah, posts don't have title if I'm not mistaken. Topics are the thing that has the title and contains posts.

**Comment by nottrobin** [comment]
Why are you creating a JSON endpoint? Isn't it much simpler just to build and serve the HTML page directly?

File: entrypoint [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-RUN_COMMAND="talisker.gunicorn.gevent webapp.app:app --bind $1 --worker-class gevent --name talisker-`hostname`"
+RUN_COMMAND="talisker.gunicorn webapp.app:app --bind $1 --worker-class sync --workers 4 --name talisker-`hostname`"
```
Comment (nottrobin) [inline entrypoint]: Why 4? I think we now use 8 in some places, 5 in others, and here 4. Did you copy this from somewhere?

File: requirements.txt [hunk 2]
```diff
@@ -8,10 +8,14 @@ yamlordereddictloader==0.4.0
 beautifulsoup4==4.7.1
 humanize==0.5.1
 python-dateutil==2.7.5
-gunicorn[gevent]
 talisker==0.11.1
 responses==0.10.5
 
+# TEMP FIX
```
Comment (nottrobin) [inline requirements.txt]: This can be achieved by simply putting talisker above flask in this requirements file, I believe

**Comment by jpmartinspt** [comment]
To be implemented with google search instead.

**Pull request closed without merging**